docs/en/get_started.md

```diff
@@ -1,201 +1,141 @@
-## Prerequisites
-
-- Linux or macOS (Windows is in experimental support)
-- Python 3.6+
-- PyTorch 1.3+
-- CUDA 9.2+ (If you build PyTorch from source, CUDA 9.0 is also compatible)
-- GCC 5+
-- [MMCV](https://mmcv.readthedocs.io/en/latest/#installation)
-
-Compatible MMDetection and MMCV versions are shown as below. Please install the correct version of MMCV to avoid installation issues.
-
-| MMDetection version |       MMCV version        |
-|:-------------------:|:-------------------------:|
-|       master        | mmcv-full>=1.3.17, <1.6.0 |
-|       2.24.1        | mmcv-full>=1.3.17, <1.6.0 |
-|       2.24.0        | mmcv-full>=1.3.17, <1.6.0 |
-|       2.23.0        | mmcv-full>=1.3.17, <1.5.0 |
-|       2.22.0        | mmcv-full>=1.3.17, <1.5.0 |
-|       2.21.0        | mmcv-full>=1.3.17, <1.5.0 |
-|       2.20.0        | mmcv-full>=1.3.17, <1.5.0 |
-|       2.19.1        | mmcv-full>=1.3.17, <1.5.0 |
-|       2.19.0        | mmcv-full>=1.3.17, <1.5.0 |
-|       2.18.0        | mmcv-full>=1.3.17, <1.4.0 |
-|       2.17.0        | mmcv-full>=1.3.14, <1.4.0 |
-|       2.16.0        | mmcv-full>=1.3.8, <1.4.0  |
-|       2.15.1        | mmcv-full>=1.3.8, <1.4.0  |
-|       2.15.0        | mmcv-full>=1.3.8, <1.4.0  |
-|       2.14.0        | mmcv-full>=1.3.8, <1.4.0  |
-|       2.13.0        | mmcv-full>=1.3.3, <1.4.0  |
-|       2.12.0        | mmcv-full>=1.3.3, <1.4.0  |
-|       2.11.0        | mmcv-full>=1.2.4, <1.4.0  |
-|       2.10.0        | mmcv-full>=1.2.4, <1.4.0  |
-|        2.9.0        | mmcv-full>=1.2.4, <1.4.0  |
-|        2.8.0        | mmcv-full>=1.2.4, <1.4.0  |
-|        2.7.0        | mmcv-full>=1.1.5, <1.4.0  |
-|        2.6.0        | mmcv-full>=1.1.5, <1.4.0  |
-|        2.5.0        | mmcv-full>=1.1.5, <1.4.0  |
-|        2.4.0        | mmcv-full>=1.1.1, <1.4.0  |
-|        2.3.0        |     mmcv-full==1.0.5      |
-|      2.3.0rc0       |     mmcv-full>=1.0.2      |
-|        2.2.1        |        mmcv==0.6.2        |
-|        2.2.0        |        mmcv==0.6.2        |
-|        2.1.0        |   mmcv>=0.5.9, <=0.6.1    |
-|        2.0.0        |   mmcv>=0.5.1, <=0.5.8    |
-
-**Note:** You need to run `pip uninstall mmcv` first if you have mmcv installed.
-If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
-
-## Installation
-
-### A from-scratch setup script
-
-Assuming that you already have CUDA 10.1 installed, here is a full script for setting up MMDetection with conda.
-You can refer to the step-by-step installation instructions in the next section.
+# Prerequisites
+
+In this section we desmonstrate how to prepare an environment with PyTorch.
+
+MMDetection works on Linux, Windows and macOS. It requires Python 3.6+, CUDA 9.2+ and PyTorch 1.5+.
+
+:::{note}
```

Comment:
Rendering failed

Reply:
I will switch to an alternative for better compatibility.